test/routes/routesApi.spec.js

```diff
@@ -47,6 +57,109 @@ describe('Le serveur MSS des routes /api/*', () => {
     });
   });
 
+  describe('quand requête GET sur `/api/services/export.csv`', () => {
+    beforeEach(() => {
+      testeur.adaptateurCsv().genereCsvServices = () => Promise.resolve();
+    });
+
+    it("vérifie que l'utilisateur est authentifié", (done) => {
+      testeur.middleware().verifieRequeteExigeAcceptationCGU(
+        'http://localhost:1234/api/services/export.csv',
+        done,
+      );
+    });
+
+    it('aseptise les paramètres de la requête', (done) => {
```

Comment:
Soyons plus précis : `it('aseptise les identifiants des services à exporter', (done) => {`